operator_ui/src/types/generated/graphql.d.ts

```diff
@@ -0,0 +1,363 @@
+export type Maybe<T> = T | null;
```

Comment:
Should I be committing the generated code. 

If I don't commit it, the typescript check will fail.

Reply:
We currently run generate in github actions before type checking. You would need to run generate even if you commit the schema to make sure that it is not out of date. It's better not to commit generated files (`graphql.d.ts` and `graphql.schema.json`) and always generate them.